utils/check-style/check_cpp.sh

```diff
@@ -4,35 +4,32 @@
 
 cd /ClickHouse/utils/check-style || echo -e "failure\tRepo not found" > /test_output/check_status.tsv
 
-start_total=`date +%s`
+start_total=$(date +%s)
 
-# FIXME: 30 sec to wait
-# echo "Check duplicates" | ts
-# ./check-duplicate-includes.sh |& tee /test_output/duplicate_includes_output.txt
+start=$(date +%s)
+./check-duplicate-includes.sh |& tee /test_output/duplicate_includes_output.txt
+runtime=$(($(date +%s)-start))
+echo "Check duplicates. Done. $runtime seconds."
```

Comment:
does it run much faster now? I don't think this check will provide enough benefit to be worth the tens of seconds of runtime

Reply:
Maybe it's completely redundant now since it will break on builds. But let's ask devs how important it is at all.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/61639 here I'll test if duplicates are entirely redundant in the early stage.

Also, @rschu1ze said that the duplicates would fail the clang-tidy check. It is another reason to have Builds_1 and Builds_2 run in parallel.

Reply:
https://github.com/ClickHouse/ClickHouse/actions/runs/8358525055/job/22880135656?pr=61639#step:8:11961

The clang-tidy catched it.